README.md

```
@@ -1,12 +1,50 @@
+## WSL2 Deployment in Windows 11 (Ubuntu 26.04 and AlmaLinux 10)
+
+This project fully supports and is designed for only **localhost** deployments inside a Windows 11 Linux WSL2 environment. Below are the steps to deploy WSL2 and execute the playbooks or shell scripts.
+
+### Step 1: Install WSL2 on Windows 11
+Open a Windows PowerShell terminal with **Administrator** privileges and run:
+```powershell
+# Install WSL2 with the default Ubuntu 26.04 distro
+wsl --install -d Ubuntu-26.04
+```
+Alternatively, if you want to deploy **AlmaLinux 10**, you can download the AlmaLinux 10 WSL appx/zip package from the official AlmaLinux channels or import it:
+```powershell
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)